OpenImis.ePayment/Extensions/GepgFileLogger.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
```

Comment:
why this is not part of the escape folder ? if it is not not related to GePG then it should be renamed